Jessy/stations_challenge.py

```diff
@@ -0,0 +1,127 @@
+'''
+NYC Transit Challenge! 
+
+In this challenge, you will use OOP and inheritance to design subway and bus stations!
+You'll use the parent class Station to make child classes for SubwayStation and BusStation. 
+
+Note, you should NOT need to change the Station class at all in this challenge.
+
+Since subways and buses have different information, the methods and attributes will be a bit different
+'''
+
+
+print('Question 1: Making the SubwayStation Class')
+'''
+Using the Station class below as the parent, make a child class called SubwayStation
+
+SubwayStation should:
+-have an additional attribute called 'lines' that is user-defined as a list during initialization. 
+    this will indicate which subway lines stop at the station (for example ['A', 'C'])
+-override the show_info() method from Station to display which subway lines stop there, in addition to the station_name and location
+'''
+class Station:
+    def __init__(self, station_name, location):
+        self.station_name = station_name
+        self.location = location
+    
+    def show_info(self):
+        print(f'{self.station_name} station is located at {self.location}')
+
+#  I put the parent inside the parathensis so the child can inherant its attributes.
+
+class SubwayStation(Station):
+    def __init__(self, station_name, location, lines):
+# used super() so I wont need to type self again
+        super().__init__(station_name, location)
+        self.lines = lines
+# I instantiated the information for my s1
+# s1= SubwayStation('14th street', '14th street and 7th avenue', ['1','2', '3', 'L'])
```

Comment:
Great note if this helps you keep track of the info entered into print statement